bokeh/plotting/figure.py

```diff
@@ -150,12 +150,6 @@ class Figure(Plot):
     __view_model__ = "Plot"
 
     def __init__(self, *arg, **kw):
-
-        if 'plot_width' in kw and 'width' in kw:
-            raise ValueError("Figure called with both 'plot_width' and 'width' supplied, supply only one")
-        if 'plot_height' in kw and 'height' in kw:
-            raise ValueError("Figure called with both 'plot_height' and 'height' supplied, supply only one")
```

Comment:
What happens if both are passed now?

Reply:
This is well defined. Initialization happens in the order of iteration of the keyword arguments, so whatever comes last. I removed this, because it seems unnecessary and may interfere with cloning. Providing both is redundant, but technically correct, the same as you can have `Plot(width=100, plot_height=100)`.